src/NMEA/ExternalSettings.cpp

```diff
@@ -22,6 +22,7 @@ Copyright_License {
 */
 
 #include "ExternalSettings.hpp"
+#include "LogFile.hpp"
```

Comment:
No no, don't add useless log messages everywhere!

Reply:
All logs removed, except one when a wrong protocol element is received.